internal/driver/gomobile/window.go

```diff
@@ -130,6 +135,11 @@ func (w *window) Hide() {
 }
 
 func (w *window) Close() {
+	if w.onCloseIntercepted != nil {
```

Comment:
As in the GLFW this creates a loop. The close button on the mobile driver windows would have to call a different function to trigger the close intercept. I don't think that needs to be exposed as this is all in the same package.